app/views/help/cookie_settings.html.erb

```diff
@@ -19,6 +19,7 @@
       } do %>
         <p><%= t('cookies.explanation') %></p>
         <p><%= t('cookies.how_we_use') %></p>
+        <p><%= t('cookies.explanation_html') %></p>
```

Comment:
I noticed the indentation is different from the original PR, not sure if this is intended

Reply:
Yes, I think this matches up with recent changes 👍 shouldn't affect how the page is rendered though.